hs-bindgen/src-internal/HsBindgen/Frontend/Pass/MangleNames.hs

```diff
@@ -59,9 +61,25 @@ mangleNames unit =
 type NameMap = Map (C.QualId HandleTypedefs) HsIdentifier
 
 data MangleError =
-    CouldNotMangle Text
+    CouldNotMangle !Text
+  | MissingDeclaration !(C.QualId HandleTypedefs)
   deriving stock (Show, Eq)
```

Comment:
What's the purpose of the bangs?

Reply:
I was expecting you to ask that :-). It is a mixture of
- the linter telling me to put those bangs
- me programming a lot of number crunching simulations in Haskell and having serious performance problems with lazy data structures.

That's why I have the habit to make constructor fields strict.

I will remove those.